st/m0t1fs-buffsize-layout-change-test.sh

```diff
@@ -22,9 +22,9 @@
 set -e
 
 SANDBOX_DIR=${SANDBOX_DIR:-/var/motr/sandbox.m0t1fs-writesize-st}
-M0_SRC_DIR=`readlink -f $0`
+M0_SRC_DIR=`readlink -f "$0"`
```

Comment:
Please fix another warning by:
M0_SRC_DIR=$(readlink -f "$0")

Reply:
@huanghua78 As addressed by @rkothiya in PR https://github.com/Seagate/cortx-motr/pull/1892, double quoting that reference leads to failure of tests. Please let me know if I should double quote this reference or not.

Reply:
Sorry, I don't understand the above reply.

Reply:
Double quoting this ```$0``` to ```"$0"``` will result in failing of tests.